### PR TITLE
Fix Authentication state documentation

### PR DESCRIPTION
**Pull request by BerserkerTroll**
This — is how it actually works!

### CODE REVIEW

File: docs/source/reference/authenticators.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ If such state should be persisted, `.authenticate()` should return a dictionary
 
 ```python
 {
-  'username': 'name',
+  'name': username,
```
Comment (betatim) [inline docs/source/reference/authenticators.md]: can you quote `username` again? I think in the rest of this example all values are quoted to make it a valid dict.

**Comment by betatim** [comment]
LGTM except for that small nitpick on quoting.

File: docs/source/reference/authenticators.md (hunk 1 above)
Comment (BerserkerTroll) [reply docs/source/reference/authenticators.md]: Without quotes it is more clear that this is a variable and not a string literal.

**Comment by willingc** [comment]
Thanks @betatim and @BerserkerTroll

**Pull request merged**